src/reducers/index.js

```diff
@@ -1234,4 +1236,34 @@ export const onBlueprintAddInformation = (state, { payload }) => {
   };
 };
 
+/**
+ * @param {Object} state
+ * @param {{
+ *   payload: {
+ *     ncId: string,
+ *     address: string,
+ *   }
+  * }} action
+ */
```

Comment:
```suggestion
/**
 * @param {Object} state
 * @param {Object} action
 * @param {string} action.payload.ncId
 * @param {string} action.payload.address
 */
```

Also, we should have an explanation of the effects of this reducer method.

Reply:
Improved the docstring https://github.com/HathorNetwork/hathor-wallet/pull/623/commits/1ef8ca11cd1729981eff0cdc85d04fc7c74de7bc